1.26/Dockerfile.fedora

```diff
@@ -27,8 +23,8 @@ LABEL summary="${SUMMARY}" \
       io.k8s.display-name="Nginx ${NGINX_VERSION}" \
       io.openshift.expose-services="8080:http" \
       io.openshift.expose-services="8443:https" \
-      io.openshift.tags="builder,${NAME},${NAME}${NGINX_SHORT_VER}" \
-      com.redhat.component="${NAME}" \
+      io.openshift.tags="builder,${NAME},${NAME}-${NGINX_SHORT_VER}" \
+      com.redhat.component="${NAME}-${NGINX_SHORT_VER}" \
```

Comment:
Is this intended to differ from the C10S entry?
`      com.redhat.component="${NAME}-${NGINX_SHORT_VER}-container" \`
vs
`      com.redhat.component="${NAME}-${NGINX_SHORT_VER}" \`